src/App.vue

```diff
@@ -7,13 +7,106 @@
 </template>
 
 <script>
+import { workflowService } from 'workflow-service'
+
 const defaultLayout = 'empty'
 
+const QUERIES = {
```

Comment:
Why is this stuff here rather than in `GScan.vue`?

Reply:
I got used to having a component rendering, and the data fetching/massaging elsewhere. As it seems to make the components easier to maintain.

So for `GScan`, I tried to create it with only props, and then plug the data in. That's when I realized I didn't have a view/route for GScan, as it is displayed in pretty much every view of the system. It is also easier to write unit tests and check the coverage.

But if you find it confusing or that it is not how it was supposed to work with the WorkflowService, we can move it to `GScan.vue`. As that component it used throughout the application maybe it makes sense that it both fetches data and also renders it.

Reply:
Moving it to `GScan.vue`. If necessary we can move it to somewhere else later 👍